roles/dtc/common/templates/ndfc_vrf_lite.j2

```diff
@@ -1,6 +1,13 @@
 {% if item.ospf is defined and item.ospf %}
 {% include 'ndfc_vrf_lite_ospf.j2' %}
 {% endif %}
+{% set bgp = {} %}
 {% if item.bgp is defined and item.bgp %}
-{% include 'ndfc_vrf_lite_ebgp.j2' %}
-{% endif %}
\ No newline at end of file
+  {% set _ = bgp.update({'enabled': true}) %}
```

Comment:
Indentation looks odd here.  Can we keep it consistent?  Looks like in most places we use 2 spaces, not 4.

Reply:
it was a typo, thanks for reviewing